src/formio/components/radio.ts

```diff
@@ -0,0 +1,39 @@
+import {InputComponentSchema} from '..';
+import {OFExtensions} from '../base';
+import {ManualValues, Option, VariableValues} from '../common';
+
+type Validator = 'required';
+type TranslatableKeys = 'label' | 'description' | 'tooltip';
+
+export type RadioInputSchema = InputComponentSchema<string | null, Validator, TranslatableKeys>;
+
+/**
+ * @group Form.io components
+ * @category Base types
+ */
+interface RadioManualValuesSchema extends Omit<RadioInputSchema, 'hideLabel' | 'disabled'> {
+  type: 'radio';
+  defaultValue?: string | null;
```

Comment:
can we make the key required?

```suggestion
  defaultValue: string | null;
```